grails-app/domain/org/pih/warehouse/order/OrderSummary.groovy

```diff
@@ -29,6 +29,7 @@ class OrderSummary {
     static mapping = {
         version false
         cache usage: "read-only"
+        table "mv_order_summary"
```

Comment:
order_summary_mv

Although I think we should take another look at our naming conventions. Part of me is thinking that I'd like this one to be order_summary and we can rename the view to be order_summary_v or something like that. But we can leave that discussion for another time.

Reply:
done